examples/interactive_bluefog_helloworld.ipynb

```diff
@@ -0,0 +1,683 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running BlueFog with jupyter notebook\n",
+    "\n",
+    "The **goal** of this ipython notebook is to show you how you can run BlueFog code in ipython notebook environment.\n",
+    "\n",
+    "As you know, the BlueFog framework is aimed to run decentralized algorithm over a cluster of processes or machines.\n",
+    "Meanwhile, jupyter notebook is an execellent interactive frontend talking with single IPython process. The contradiction between a single frontend process and a cluster of backend processescan be completely circumvented, fortunately. The main idea is using the jupyter notebook as the driver to inform how the cluster will execute the BlueFog code. On one hand, the execution of BlueFog still runs over multiple processes so that you can enjoy the scalability and parallelism brought by BlueFog. On the other hand, you (the programmer) can interactively execute with BlueFog and monitor and control the code in the single place.\n",
+    "\n",
+    "## Prerequisites: \n",
+    "To begin with, make sure that you have installed OpenMPI and BlueFog in your system first.\n",
+    "Besides these, you also need to install [ipyparallel](https://ipyparallel.readthedocs.io/en/latest/index.html) package for managering the parallel computation.\n",
+    "Usually, you can install it through `pip install jupyter ipyparallel`.\n",
+    "\n",
+    "## Steps to run Bluefog with Jupyter Notebook\n",
+    "\n",
+    "- **Step 1: Start jupyter notebook**\n",
+    "\n",
+    "```shell\n",
+    "jupyter notebook\n",
+    "```\n",
+    "\n",
+    "- **Step 2A: Start the `ibfrun` (interactive bluefog run) in a single machine**\n",
+    "\n",
+    "```shell\n",
+    "ibfrun start -np 4\n",
```

Comment:
this code seems not working?

ibfrun -np 4 start works.

Reply:
I updated the command usage. The lastest version is ibfrun start -np 4
instead.